uk.co.compucorp.civicrm.hrleaveandabsences/xml/schema/CRM/HRLeaveAndAbsences/LeaveRequestCalendarFeedConfig.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="iso-8859-1" ?>
+
+<table>
+  <base>CRM/HRLeaveAndAbsences</base>
+  <class>LeaveRequestCalendarFeedConfig</class>
+  <name>civicrm_hrleaveandabsences_calendar_feed_config</name>
+  <comment>Leave Request Calendar Feed Config</comment>
+  <add>4.4</add>
+  <log>true</log>
+
+  <field>
+    <name>id</name>
+    <type>int unsigned</type>
+    <required>true</required>
+    <comment>Unique LeaveRequestCalendarFeedConfig ID</comment>
+    <add>4.4</add>
+  </field>
+  <primaryKey>
+    <name>id</name>
+    <autoincrement>true</autoincrement>
+  </primaryKey>
+
+  <field>
+    <name>title</name>
+    <type>varchar</type>
+    <length>127</length>
+    <required>true</required>
+    <unique>true</unique>
+    <comment>The Calendar Feed Config Title. There cannot be more than one entity with the same title</comment>
+  </field>
+
+  <field>
+    <name>timezone</name>
+    <type>varchar</type>
+    <length>50</length>
+    <required>true</required>
+    <comment>The Calendar Feed Config Timezone</comment>
+  </field>
+
+  <field>
+    <name>hash</name>
+    <type>varchar</type>
+    <length>50</length>
+    <required>true</required>
+    <unique>true</unique>
+    <comment>The Calendar Feed Config Hash. Should be unique per config</comment>
+  </field>
+
+  <field>
+    <name>is_active</name>
+    <type>boolean</type>
+    <default>1</default>
+    <comment>Whether the feed is active or not</comment>
```

Comment:
Maybe this should be required too?

Reply:
It has a default value though, thats why i made it not to be required